tests/integration/defs/examples/test_whisper.py

```
@@ -136,3 +136,68 @@ def test_llm_whisper_general(llm_venv, engine_dir, data_type,
+            build_cmd.append("--max_input_len=3000")
+            build_cmd.append("--max_seq_len=3000")
+        if component == "decoder":
+            build_cmd.append("--max_input_len=14")
+            build_cmd.append("--max_seq_len=114")
+            build_cmd.append("--max_encoder_input_len=3000")
+            build_cmd.append(f"--max_beam_width={num_beams}")
+            build_cmd.append("--gemm_plugin=float16")
+            build_cmd.append("--bert_attention_plugin=float16")
+            build_cmd.append("--gpt_attention_plugin=float16")
+
+        check_call(" ".join(build_cmd), shell=True, env=llm_venv._new_env)
```

Ruff (0.15.6)

[error] Function call with `shell=True` parameter identified, security issue

(S604)